skipruntime-ts/ts/src/skipruntime_api.ts

```diff
@@ -16,6 +16,8 @@ export type TTableCollection = any;
 export type TTable = any;
 export type Param = any;
 
+export type RefreshToken = number & { __type_brand: "SkipRefreshToken" };
```

Comment:
The implementation of `freeze` does `Object.defineProperty(value, "__sk_frozen"` which seems to be playing a similar role. Maybe it is worthwhile to use just one mechanism in both places?

Reply:
No, here, it's a hack for opaque types, it doesn't convey any value.

Feel free to use `Internal.Opaque<number, "SkipRefreshToken">`